### PR TITLE
Alerting hipchat

### PR DESCRIPTION
**Pull request by jregovic**
https://github.com/grafana/grafana/issues/6451

Adds an alert notification using the HipChat v2 room API.
Messages go to the v2/room/ROOMID/notification?auth_token=AUTHTOKEN endpoint

### CODE REVIEW

**Comment by torkelo** [comment]
this needs rebase :)

**Comment by bergquist** [comment]
master have now been merged into grafana/alerting_hipchat.

Please rebase your changes from grafana/alerting_hipchat if you want to target alerting_hipchat.

**Comment by torkelo** [comment]
what is the alerting_hipchat branch?

**Comment by jregovic** [comment]
alerting_hipchat is the branch created for https://github.com/grafana/grafana/issues/6451. My changes add the hook to HipChat.

**Comment by bergquist** [comment]
@torkelo it was requested the make collaboration easier. 

This PR still need to rebase against the `alerting_hipchat` branch.

**Comment by jregovic** [comment]
I'm pretty sure I've rebased this correctly. Am I missing something?

File: pkg/services/alerting/notifiers/hipchat_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+				not, err := NewHipChatNotifier(model)
+				hipchatNotifier := not.(*HipChatNotifier)
+
+				So(err, ShouldBeNil)
+				So(hipchatNotifier.Name, ShouldEqual, "ops")
+				So(hipchatNotifier.Type, ShouldEqual, "hipchat")
+				So(hipchatNotifier.Url, ShouldEqual, "http://google.com")
+				So(hipchatNotifier.ApiKey, ShouldEqual, "")
+				So(hipchatNotifier.RommId, ShouldEqual, "")
```
Comment (bergquist) [inline pkg/services/alerting/notifiers/hipchat_test.go]: Typo. This doesn't compile.

**Review by bergquist — CHANGES_REQUESTED** [review]
The commit author have to be the same as the github user. Otherwise you cannot sign the CLA which is required before we merge any PR

File: pkg/services/alerting/notifiers/hipchat_test.go (hunk 1 above)
Comment (jregovic) [inline pkg/services/alerting/notifiers/hipchat_test.go]: fixed.

**Comment by bergquist** [comment]
This is starting to get out of hand :) I suggest you move the code for the alerting notifier to a new branch and create a new pull request that targets the master branch. 

The hipchat branch was created to make collaboeration with others easier but since its only you working on it its much easier to just target the master branch.

Closing this since 97 files changes is way to much to review. But please open a new pull request!

**Pull request closed without merging**